packages/xstate-test/src/TestModel.ts

```diff
@@ -83,9 +83,9 @@ export class TestModel<TState, TEvent extends EventObject> {
   }
 
   public getPlans(
-    planGenerator: PlanGenerator<TState, TEvent>,
-    options?: Partial<TraversalOptions<TState, TEvent>>
+    options?: GetPlansOptions<TState, TEvent>
```

Comment:
`getPlans` now takes the `planGenerator` from options, instead of a required first argument.